x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -193,47 +212,23 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
             exists ? 'exists' : 'missing'
           }`}
         >
-          <EuiKeyboardAccessible>
```

Comment:
Was it intentional to remove this? By removing it, you've changed the behavior: it is no longer possible to navigate this list of fields with the `tab` key because `tabindex` was set by EuiKeyboardAccessible.

Reply:
Hi @wylieconlon, double checking if this is what you mean by navigating the list of fields using tab?
![image](https://user-images.githubusercontent.com/4016496/89453675-79bd7a80-d714-11ea-8d48-f382c55261c7.png)

Removing `EuiKeyboardAccesible` was intentional. @myasonik made that change (and other a11y improvements) on [this](https://github.com/andreadelrio/kibana/pull/2) PR, if you want more details.

Reply:
@wylieconlon, the element now returns an actual `<button>` element instead of a `<div>` with an `onClick` handler making this component unnecessary.